### PR TITLE
Tasnhuba

### PR DESCRIPTION
**Pull request by tasnhubaislam**

### CODE REVIEW

**Review by joerossi215 — APPROVED** [review]
Fantastic!!!

**Review by yellow-tulips — APPROVED** [review]
All of these files look good to go.

File: index.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                <div id="quote"> 
+                    <div id="output">Get your quote of the day!</div>
+                    <button onclick="quotes()" id = "button"> click me!<br></button>
+                </div>
+            </div>
+
+
+            <div class = "goalTracker">
+                <div class = "tracker">
+                    <h2>Goal Tracker</h2>
```
Comment (Taraque-A) [inline index.php:124]: Looks really good!

File: login.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<?php 
+    session_start();
```
Comment (Taraque-A) [inline login.php:2]: Nice job!

File: register.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+      }
+    }
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Register</title>
```
Comment (Taraque-A) [inline register.php:65]: Great work!

**Review by Taraque-A — APPROVED** [review]

**Review by farzanaisrat — APPROVED** [review]
looks good!

**Pull request merged**